.github/workflow/auto-author-assign.yml

```
@@ -0,0 +1,12 @@
+name: 'Auto Author Assign'
+on:
+  pull_request:
+    types: [opened, reopened]
+jobs:
+  add-assignees:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Test toshimaru/auto-author-assign action
```

Can we change that to a different name or it has to be this specific one?

---

Yes, you can put whatever you want in the `-name` tag

---

Then let's change it to something less bloated.

```suggestion
      - name: auto-author-assign
```